runtimes/spiritnet/src/lib.rs

```diff
@@ -123,16 +123,14 @@ impl frame_system::Config for Runtime {
 	/// The lookup mechanism to get account ID from whatever is passed in
 	/// dispatchers.
 	type Lookup = AccountIdLookup<AccountId, ()>;
-	/// The index type for storing how many extrinsics an account has signed.
-	type Index = Index;
-	/// The index type for blocks.
-	type BlockNumber = BlockNumber;
+	/// The nonce type for storing how many extrinsics an account has signed.
+	type Nonce = runtime_common::Nonce;
```

Comment:
Can't you bring `use runtime_common::Nonce`?

Reply:
I prefer the runtime_common::Nonce declaration because it makes it clearer where the value is defined, and the code remains clean. But updated in: [94b4a70](https://github.com/KILTprotocol/kilt-node/pull/559/commits/94b4a70489d1a4fa1503e9729089c2036be4ce4d)